test/unit-test.js

```diff
@@ -0,0 +1,46 @@
+const { assert, expect } = require('chai')
+const fs = require('fs');
+const h5 = require('../index');
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+
+const filePath = __dirname + '/Acoustic-Breeze.mp3';
```

Comment:
I wonder if we could just mock this rather than making everyone download this file. Easier said than done based on the fact you're using the specific values in the test. ¯\_(ツ)_/¯ But it's definitely something I would try to do.